src/test/components/ListOfPublishedProfiles.test.js

```diff
@@ -339,65 +343,60 @@ describe('ListOfPublishedProfiles', () => {
       const endpointUrl = `https://api.profiler.firefox.com/profile/${profileToken}`;
       mockFetchForDeleteProfile({ endpointUrl, jwtToken });
 
-      jest.useFakeTimers(); // ButtonWithPanel has some asynchronous behavior.
```

Comment:
as I explained, I removed all uses of `useFakeTimers` because they don't work well with the library fake-indexeddb. I hope we won't have to use both together in the future :/